gnomad_qc/v5/annotations/compute_coverage.py

```diff
@@ -0,0 +1,1054 @@
+                    overwrite=overwrite,
+                )
+
+        if args.compute_all_cov_release_stats_ht:
+            logger.info(
+                "Computing coverage, all sites allele number, and optionally quality histograms HT for %s...",
+                project,
+            )
+
+            # Context Table is used because it contains every locus in the GRCh38
+            # reference as opposed to a ref-blocked VDS reference dataset.
+            ref_ht = vep_context.versions["105"].ht()
+            if test_chr22_chrx_chry:
+                ref_ht = hl.filter_intervals(
+                    ref_ht, [hl.parse_locus_interval(c) for c in chrom]
+                )
+            elif test_2_partitions:
+                ref_ht = ref_ht._filter_partitions(range(2))
+
+            # Retain only 'locus' annotation from context Table.
+            ref_ht = ref_ht.key_by("locus").select().distinct()
+
+            # The AoU dataset should have removed centromeres and telomeres, but
+            # this serves as an additional safeguard.
+            ref_ht = hl.filter_intervals(
+                ref_ht,
+                telomeres_and_centromeres.ht().interval.collect(),
+                keep=False,
+            )
+            ref_ht = ref_ht.checkpoint(hl.utils.new_temp_file("ref", "ht"))
+
+            sex_karyotype_field = "meta.sex_imputation.sex_karyotype"
+            if project == "aou":
+                vds = get_aou_vds(
+                    release_only=True,
+                    filter_partitions=range(2) if test_2_partitions else None,
+                    annotate_meta=True,
+                    chrom=["chr22", "chrX", "chrY"] if test_chr22_chrx_chry else None,
+                )
+                # NOTE: AoU v8 VDS does not have DP annotation, so using custom function
+                # to annotate adj.
+                # Also adding DP here to make sure `compute_stats_per_ref_site`
+                # doesn't throw an error.
+                vmt = vds.variant_data
+                vmt = annotate_adj(vmt)
+                vmt = vmt.annotate_entries(DP=hl.fold(lambda x, y: x + y, 0, vmt.LAD))
+                vds = hl.vds.VariantDataset(vds.reference_data, vmt)
+
+                if test:
+                    meta_ht = meta_ht.filter(
+                        (meta_ht.project_meta.project == project) & (meta_ht.release)
+                    ).select()
+                    meta_ht = meta_ht.sample(0.001)
+                    vds = hl.vds.filter_samples(vds, meta_ht)
+            else:
+                vds = get_gnomad_v5_genomes_vds(
+                    release=True,
+                    consent_drop=True,
+                    filter_partitions=range(2) if test_2_partitions else None,
+                    annotate_meta=True,
+                    chrom=["chr22", "chrX", "chrY"] if test_chr22_chrx_chry else None,
+                )
+
+            validate_vds(vds)
+            check_resource_existence(
```

Comment:
should this check be moved up to be right below  L734
"if args.compute_all_cov_release_stats_ht:"?